### PR TITLE
PRISB-351: Homepage heatmap tweaks

### PR DESCRIPTION
**Pull request by rpeterman-gp**
## [PRISB-351: Homepage design tweaks based on heatmap results](https://fourkitchens.atlassian.net/browse/PRISB-351)

**This PR does the following:**
- Teaks to homepage card item layout
- Tweak to icon alignment in header
- Adds Viewport Storybook addon
- Upgrades Storybook to v3.4.8

### To Review:

- [x] Checkout this branch.
- [x] Run `yarn && yarn start`.
- [x] Got to [Molecules > CarItem > Default](http://localhost:9001/?selectedKind=Molecules%2FCardItem&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel).
- [x] Used __Viewport > Device__ to view card at various device sizes.
- [x] Ensure car appears as it should on phone, tablet and desktop sizes.
- [x] Repeat for [Large](http://localhost:9001/?selectedKind=Molecules%2FCardItem&selectedStory=Large&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook-addon-viewport%2Faddon-panel) and [Freeform](http://localhost:9001/?selectedKind=Molecules%2FCardItem&selectedStory=Freeform&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook-addon-viewport%2Faddon-panel) stories.
- [x] Go to [Organisms > Header > Default](http://localhost:9001/?selectedKind=Organisms%2FHeader&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook-addon-viewport%2Faddon-panel).
- [x] Ensure icons align center with logo at all device sizes.
- [x] Go to [Pages > Home > Default](http://localhost:9001/?selectedKind=Pages%2FHome&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook-addon-viewport%2Faddon-panel).
- [x] Ensure card items look as expected in use.

### CODE REVIEW

File: src/components/Molecules/CardItem/CardItem.css [hunk 1]
```diff
@@ -117,7 +108,71 @@ div.blurb {
...
+    grid-area: IMAGE;
+  }
+
+  .cardItem.normal .titleWrap {
+    grid-area: TITLE;
+    align-self: center;
+  }
+
+  .cardItem.normal .title {
+    font-size: 1.2rem;
```
Comment (brandonhundt) [inline src/components/Molecules/CardItem/CardItem.css]: @rpeterman-gp After reviewing this on my screen, this still feels quite big, especially with the image next to it on mobile. I think `1.1rem` would do the trick.

File: src/components/Molecules/CardItem/CardItem.css [hunk 2]
```diff
@@ -3,96 +3,87 @@
...
-.link:hover {
-  color: orange;
+.cardItem.normal .titleWrap {
+  margin: 0;
+  padding: 0;
 }
 
-.image {
+.title {
+  font-size: 1.25rem;
```
Comment (brandonhundt) [inline src/components/Molecules/CardItem/CardItem.css]: for consistency this should match remaining card items. 1.1rem.

File: src/components/Molecules/CardItem/CardItem.css [hunk 3]
```diff
@@ -117,7 +108,71 @@ div.blurb {
...
+  .cardItem.normal .titleWrap {
+    align-self: end;
+  }
+
+  .title {
+    font-size: 1.5rem;
+  }
+
+  .blurb {
+    display: block;
```
Comment (brandonhundt) [inline src/components/Molecules/CardItem/CardItem.css]: `line-height: 1.3;` Decrease the line-height slightly. Doubt this is the right place for this style.

Comment (rpeterman-gp) [reply src/components/Molecules/CardItem/CardItem.css]: @brandonhundt Do you want the line height to be `1.3` down from `1.5` globally?

Comment (brandonhundt) [reply src/components/Molecules/CardItem/CardItem.css]: Just for the teaser, I want the line-height to be 1.3. Good follow up

Comment (rpeterman-gp) [reply src/components/Molecules/CardItem/CardItem.css]: Should this line height apply to freeform content or just teaser blurb?

Comment (brandonhundt) [reply src/components/Molecules/CardItem/CardItem.css]: Freeform would make sense as well.

**Pull request merged**